CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fixed cases where description was used with non-error status code when creating Status objects.
   ([#504](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/504))
+  ### Changed
+- Added optional `schema_url` field to `Resource` class
+  ([opentelemetry-python #1862](https://github.com/open-telemetry/opentelemetry-python/issues/1862))
```

Comment:
Copied from my comment in the [core PR](https://github.com/open-o11y/opentelemetry-python/pull/23/files#r638846672)
> By the way, this should be the number and link to the PR you submit on the upstream repo. Since you usually wouldn't have submitted it yet, you could just put TBD here until you submit to upstream. Or, you could also use the one you accidentally opened/closed and re-open that PR 😄